modules/unsupported/mbtiles/pom.xml

```diff
@@ -39,26 +39,22 @@
   </licenses>
 
 
-  <!-- =========================================================== -->
-  <!--     Developers and Contributors                             -->
-  <!-- =========================================================== -->
-  <developers>
-    <developer>
-      <id>YOURID</id>
-      <name>YOUR NAME</name>
-      <email>you@server.org</email>
-      <organization>University, Organization or Company</organization>
-      <roles>
-        <role>Module Maintainer</role>
-        <role>Java Developer</role>
-      </roles>
-    </developer>
-  </developers>
-
-
   <!-- =========================================================== -->
   <!--     Dependency Mangement                                    -->
   <!-- =========================================================== -->
+  <repositories>
+    <repository>
+      <id>ECC</id>
+      <url>https://maven.ecc.no/releases</url>
```

Comment:
Please do not add additional repositories to the build, we can setup osgeo repo to cache this repository so that downstream projects to not need to maintain a wide range of repositories to check (which vastly slows down their build process)

I understand that this is an unsupported module, so you can put off this feedback until the module is ready for release.

Reply:
update: I have added `no.ecc-cache` to repo.osgeo.org and included it `repo.osgeo.org/repository/release`.

I was concerned on your use of this for both building your own projects and publishing new releases of vector tiles (I can see other dependencies brought into https://maven.ecc.no/releases).

As an alternative (if this is too heavy handed) you should have ability to upload third-party-dependences to osgeo repo as per geotools developer guide.